### PR TITLE
adding D to yaw

### PR DESCRIPTION
**Pull request by orneryd**
Assuming proper filtering, is there any real reason to not include D on yaw? D on yw helps with propwash considerably.

### CODE REVIEW

**Comment by kidBrazil** [comment]
Seems travis is mad at you @orneryd  - https://travis-ci.org/ButterFlight/butterflight/jobs/360904924

File: src/main/flight/pid.c [hunk 1]
```diff
@@ -104,11 +104,11 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .pidSumLimit = PIDSUM_LIMIT,
         .pidSumLimitYaw = PIDSUM_LIMIT_YAW,
         .yaw_lpf_hz = 0,
-        .dterm_lpf_hz = 100,    // filtering ON by default
+        .dterm_lpf_hz = 65,    // filtering ON by default
```
[the remark this answers is not in the document]
Comment (kidBrazil) [reply src/main/flight/pid.c]: Hey @orneryd Can you please drop Yaw D to a default of 10 - would be safer for users. We can instruct them to tune it.

[the remark this answers is not in the document]
Comment (orneryd) [reply src/main/flight/pid.c]: yep actually I set it to 8

**Comment by orneryd** [comment]
@kidBrazil default d on yaw is 8. we tested 20, 10, and 5 defaults. setting it to 0 will make it feel like betaflight. * seemed to be enough to really tell it made a significant different but without it being a jarring experience.

**Comment by orneryd** [comment]
closing this in favor of the odin branch.

**Pull request closed without merging**